integration/predata_relations_queries_test.go

```diff
@@ -190,6 +190,42 @@ PARTITION BY LIST (gender)
 			Expect(tables).To(HaveLen(1))
 			structmatcher.ExpectStructsToMatchIncluding(&tableFoo, &tables[0], "Name", "Schema")
 		})
+		It("ignore tables those parents is in extension", func() {
+			testhelper.AssertQueryRuns(connectionPool,
+				`create table public.t_part (a int, b int, c text) partition by range (b) (start (1) end (3) every (1));`)
+			testhelper.AssertQueryRuns(connectionPool,
+				`create table public.t_part_e (a int, b int, c text) partition by range (b) (start (1) end (3) every (1));`)
+			testhelper.AssertQueryRuns(connectionPool, `set allow_system_table_mods=true;`)
+			testhelper.AssertQueryRuns(connectionPool,
+				`insert into pg_depend values(1259, 'public.t_part_e'::regclass::oid, 0, 'pg_extension'::regclass::oid, 123, 0, 'e');`)
```

Comment:
I think we should use `create extension ...` with tables in tests, instead inserting into pg_depend

Reply:
it doesn't look like any resources were used in integration tests. I don't know if there is any built-in extension that creates partitioned tables.

Reply:
> it doesn't look like any resources were used in integration tests. I don't know if there is any built-in extension that creates partitioned tables.

I added them by https://github.com/arenadata/gpbackup/pull/46

Reply:
But they are in end2end tests. How good is it to use them in integration tests?

Reply:
Why not move tests to end2end?